src/components/views/elements/FacePile.tsx

```diff
@@ -21,7 +21,7 @@ import { AvatarStack, Tooltip } from "@vector-im/compound-web";
 import MemberAvatar from "../avatars/MemberAvatar";
 import AccessibleButton, { ButtonEvent } from "./AccessibleButton";
 
-interface IProps extends HTMLAttributes<HTMLSpanElement> {
+interface IProps extends Omit<HTMLAttributes<HTMLDivElement>, "onChange"> {
```

Comment:
It would be good to document that props other than those listed are passed through to the `AccessibleButton` for the FacePile as a whole.

Also, why is `onChange` excluded?

Reply:
it breaks the types, `ChangeEvent<Element>` it not compatible with `ChangeEvent<HTMLDivElement>` or similar, its not something I wanted to rewrite AccessibleButton's types to fix